crates/transaction-emitter-lib/src/emitter/mod.rs

```diff
@@ -615,17 +647,23 @@ impl TxnEmitter {
         );
 
         let all_start_sleep_durations = mode_params.get_all_start_sleep_durations(self.from_rng());
-        let mut all_accounts_iter = all_accounts.into_iter();
-        let mut workers = vec![];
+
+        // Creating workers is slow with many workers (TODO check why)
+        // so we create them all first, before starting them - so they start at the right time for
+        // traffic pattern to be correct.
```

Comment:
it takes 5 minutes to go through the loop below, in the graceful overload case (https://aptoslabs.grafana.net/goto/GaWX1nu4g?orgId=1) with 180k workers to create, which makes no sense - but at least with this change it doesn't affect the measurements.